src/components/TxBuilderAttributes.js

```diff
@@ -9,17 +10,29 @@ import TimeBoundsPicker from './FormComponents/TimeBoundsPicker';
 import {connect} from 'react-redux';
 import {StrKey} from 'stellar-sdk';
 import NETWORK from '../constants/network';
-import {fetchSequence, fetchBaseFee} from '../actions/transactionBuilder';
+import {fetchSequence, fetchBaseFee, updateTxType, updateFeeBumpAttribute} from '../actions/transactionBuilder';
+import TransactionImporter from './TransactionImporter';
+import TX_TYPES from '../constants/transaction_types';
 
 function TxBuilderAttributes(props) {
-  let {onUpdate, attributes, horizonURL} = props;
+  let {onUpdate, attributes, horizonURL, dispatch, feeBumpAttributes, networkPassphrase} = props;
+  const { txType, network } = props.state;
 
   useEffect(() => {
-    props.dispatch(fetchBaseFee(horizonURL))
+    dispatch(fetchBaseFee(horizonURL))
   }, [])
   
   return <div className="TransactionAttributes">
     <div className="TransactionOp__config TransactionOpConfig optionsTable">
+
+      <OptionsTablePair label={<span>Transaction Type <HelpMark href="https://developers.stellar.org/docs/glossary/fee-bumps" /></span>}>
+        <TxTypePicker
+          value={txType}
+          onUpdate={(value) => {dispatch(updateTxType(value))}}
+        />
+      </OptionsTablePair>
+      {txType === TX_TYPES.REGULAR &&
+      <React.Fragment>
       <OptionsTablePair label={<span>Source Account <HelpMark href="https://www.stellar.org/developers/guides/concepts/accounts.html" /></span>}>
```

Comment:
note: helpmark is going to beta docs, ok?

Reply:
I'm working on updating all the links to docs here, fine to leave it